build.sbt

```
@@ -83,41 +83,13 @@ ThisBuild / headerLicense  := Some(HeaderLicense.Custom(
      |""".stripMargin
 ))
 
-// to be locked down sometime around the time of 2.13.0-RC1
-Global / mimaReferenceVersion := Some("2.13.0-RC1")
+Global / mimaReferenceVersion := None // TODO: make it Some("2.13.0-RC2") when its out
```

the smallest of nits

```suggestion
Global / mimaReferenceVersion := None // TODO: make it Some("2.13.0-RC2") when it's out
```